.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,29 @@
+<!-- Thank you for contributing a pull request! Here are a few tips to help you:
+
+1. If this is your first contribution, make sure you've read the Contribution Checklist https://developers.mattermost.com/contribute/getting-started/contribution-checklist/
+2. Read our blog post about "Submitting Great PRs" https://developers.mattermost.com/blog/2019-01-24-submitting-great-prs
+3. Take a look at other repository specific documentation at https://developers.mattermost.com/contribute
+-->
+
+#### Summary
+<!--
+A description of what this pull request does.
+-->
+
+#### Ticket Link
+<!--
+If this pull request addresses a Help Wanted ticket, please link the relevant GitHub issue, e.g.
+
+  Fixes https://github.com/mattermost/mattermost-server/issues/XXXXX
+
+Otherwise, link the Jira ticket.
+-->
+
+#### Checklist
+<!--
+Please review each item and check the ones applicable for your pull request
+-->
+- [ ] Tested related changes <!-- See https://developers.mattermost.com/contribute/server/developer-workflow/#running-only-specific-server-unit-tests how to run specific unit tests. -->
```

Comment:
I don't recall the specifics of the conversation on the developers channel, but I'm wondering if we need this line. If this particular documentation item is important enough to be called out here, I'm thinking we should just promote it somewhere on https://developers.mattermost.com/contribute/getting-started/contribution-checklist/ to make sure it gets read first.

It's also a bit confusing have a `Tested related changes` and `Added or updated unit tests`, since -- at the moment -- they amount to roughly the same things.

Reply:
My thought was that I wanted to avoide the situation where someone submits a PR that doesn't even build. But maybe this a bit to much to add this item just for these few PR's.